packages/common/src/intl/locales/en/catalogue.json

```diff
@@ -31,4 +30,4 @@
   "title.packaging": "Packaging",
   "title.pricing": "Pricing",
   "title.stock-on-hand": "Stock on Hand"
-}
+}
```

Comment:
I wonder if weblate will add the line break back in and cause a commit war? 🤔

Reply:
I've made a note to the developer about that, should be fixed in upcoming release.